src/Entity/DatasetSubmission.php

```diff
@@ -2694,4 +2682,19 @@ public function setRemotelyHostedUrl(?string $remotelyHostedUrl) : void
     {
         $this->remotelyHostedUrl = $remotelyHostedUrl;
     }
+
+    /**
+     * Check if dataset submission is marked as remotely hosted.
+     *
+     * @return bool
+     */
+    public function isMarkedRemotelyHosted(): bool
```

Comment:
"Marked" is a bit ambigious. It could have also said just `isRemotelyHosted' but this seem OK. Opinion?

Reply:
Dataset entity already has the same method name, I'm ok with **isRemotelyHosted**

Reply:
Was it changed?